src/main/java/seedu/address/logic/Messages.java

```diff
@@ -17,7 +17,8 @@ public class Messages {
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
-    public static final String MESSAGE_INVALID_DISPLAYED_INDEX = "The index provided is invalid";
+    public static final String MESSAGE_INVALID_DISPLAYED_INDEX = "The index provided is out of range of the displayed"
+            + " list";
```

Comment:
So what is the message displayed if someone inputs a non-numeric string for the index?

Reply:
It will be caught by this validation first https://github.com/AY2425S1-CS2103-F13-4/tp/pull/213

So the message would be "Index is not a non-zero integer".